contrib/database/sql/sql.go

```diff
@@ -186,6 +186,13 @@ func OpenDB(c driver.Connector, opts ...Option) *sql.DB {
 	if math.IsNaN(cfg.analyticsRate) {
 		cfg.analyticsRate = rc.analyticsRate
 	}
+	if cfg.errCheck == nil {
+		if rc.errCheck == nil {
+			cfg.errCheck = func(err error) bool { return true }
+		} else {
```

Comment:
Surely you don't need this anymore, right? That was part of the point of the `nil` check wasn't it? Or did I miss something?

Reply:
oops, you are right. just fallback to `rc.errCheck`.